mosquitto/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=mosquitto
+pkg_origin=core
+pkg_version="1.4.14"
+pkg_upstream_url="https://mosquitto.org"
+pkg_description="An Open Source MQTT v3.1/v3.1.1 Broker"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('EPL-1.0' 'EDL-1.0')
+pkg_source="http://mosquitto.org/files/source/mosquitto-${pkg_version}.tar.gz"
+pkg_shasum="156b1fa731d12baad4b8b22f7b6a8af50ba881fc711b81e9919ec103cf2942d1"
+pkg_deps=(core/glibc core/openssl nandub/c-ares core/util-linux)
+pkg_build_deps=(core/make core/gcc)
+pkg_bin_dirs=(bin)
+pkg_lib_dirs=(lib)
+
+do_build() {
+    make
+}
+
+do_install() {
```

Comment:
do_install is going to need to copy the edl-v10 and epl-v10 files into $pkg_prefix to be compliant with the licenses.

Reply:
Pfew, I'll try to read the license next time, thanks.

Reply:
This is done BTW.